doc/antora/modules/concepts/pages/modules/ldap_authentication.adoc

```diff
@@ -68,7 +68,7 @@ determine what format it is in (base64, binary, or text), and what
 password "encryption" mechanism has been used (crypt, MD5, SHA, SSHA2,
 SHA3, etc).  All that is necessary is that the
 xref:reference:raddb/mods-available/ldap.adoc[ldap module] be configured to map
-the `userPassword` LDAP field to the `&control:Password-With-Header`
+the `userPassword` LDAP field to the `control:Password-With-Header`
 attribute in FreeRADIUS.  FreeRADIUS will then "do the right thing" to
```

Comment:
Should also change ':' to '.'.  The problem with doing this later as you suggested, is you're removing a sequence of chars that more or less uniquely identifies attribute references.  It's going to be much harder to fix these are you remove the '&'.